profiles/serializers_test.py

```diff
@@ -304,3 +309,158 @@ def test_delete_employment(self):
 
         # Other profile is unaffected
         assert employment3.profile.work_history.count() == 1
+
+
+class ProfileFilledOutTests(ESTestCase):
```

Comment:
Yes, whenever a profile is created it is indexed. There is no ES functionality being tested